employees/views.py

```diff
@@ -9,10 +16,99 @@ class ReportViewSet(viewsets.ModelViewSet):
     serializer_class = ReportSerializer
     permission_classes = (
         permissions.IsAuthenticated,
-        )
+    )
 
     def get_queryset(self):
         return Report.objects.filter(author=self.request.user).order_by('-date')
 
     def perform_create(self, serializer):
         serializer.save(author=self.request.user)
+
+
+def query_as_dict(query_set):
+    dictionary = {}
+    for record in query_set:
+        key = record.date
+        dictionary.setdefault(key, [])
+        dictionary[key].append(record)
+    return dictionary
```

Comment:
I saw this function in @MartynaAnnaGottschling PR. It should be in utils and be reusable for more than one app.

Reply:
I can tell you right now, that it will change it's functionality later on to be more specific. It's used to return specific set of data, designed for rendering lists of reports in requested format. Granted such method might be *potentially* useful in other places, but in reality we store all report related views in employees app (because that's were report model and serializer are stored in the first place).

And @MartynaAnnaGottschling won't be using it anymore, because it was useless for her anyway. It also actually performed different action than mine.

Do you still want me to move it to utils? Like I said, it's a specific tool meant to be used with employees app only.